tests/user-tests.js

```diff
@@ -41,12 +41,17 @@ const TESTS = {
   withEmail: {
     payload_old: "traits({ name: 'Hello Friend', email: 'hello@friend.com' });",
     payload: "hull.traits({ name: 'Hello Friend', email: 'hello@friend.com' });",
-    result: { "email" : "hello@friend.com", "name" : "Hello Friend" }
+    result: { email: "hello@friend.com", name: "Hello Friend" }
   },
   console: {
     payload: "console.log('boom', 'bam')",
     result: {}
-  }
+  },
+  uppercase: {
+    payload_old: "traits({ 'Last_Name': 'user.traits['last_name']' })",
```

Comment:
@KgaevskiT everything looks good for me here, but I don't get only this line.
Are we setting `last_name` to string `user.traits['last_name']` instead of that parameter value? I mean the `'` quotes around `user.traits['last_name']`. It's by purpose?

I would say `'user.traits['last_name']'` will cause an syntax error due to unescaped quote.